### PR TITLE
Rating

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created a rating bar with full stars.
- created a rating bar with half stars.
Screen Shot 2022-02-17 at 12 28 07 AM
Screen Shot 2022-02-17 at 12 28 17 AM
Screen Shot 2022-02-17 at 12 28 44 AM
Screen Shot 2022-02-17 at 12 52 40 AM
Screen Shot 2022-02-17 at 12 52 17 AM

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,52 +7,202 @@
...
-      </div>
-    </div>
-    <script>
-      var modal = document.getElementById("modal");
+    <!-- full star rating -->
+
+    <!-- <div class="full-star-rating m-16 gap-3">
+      <input
+        class="full-rating-input"
+        id="star-5"
```
Comment (anshulraheja) [inline index.html:20]: I think full star rating is a part of the code. If it is not then you could just remove the commented part and refactor the code.

**Review by anshulraheja — APPROVED** [review]
Code is great and good to merge!

File: index.html (hunk 1 above)
Comment (Ananyamadhu08) [reply index.html:20]: yes, thank you!

**Review by ritikaawasthy — APPROVED** [review]
code looks great,  good to merge

**Pull request merged**